.snyk

```diff
@@ -0,0 +1,49 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+ignore:
+  "snyk:lic:golang:github.com:hashicorp:vault:sdk:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:vault:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:vault:api:auth:approle:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:vault:api:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:hcl:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:golang-lru:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-sockaddr:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-secure-stdlib:strutil:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-secure-stdlib:parseutil:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-rootcerts:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-retryablehttp:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-multierror:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-immutable-radix:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:go-cleanhttp:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+  "snyk:lic:golang:github.com:hashicorp:errwrap:MPL-2.0":
+    - "*":
+        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
+version: v1.25.0
```

Comment:
What is this version? Are we pinning to a Snyk version? Could we at least add a comment to explain what it's doing for ongoing maintenance?

Reply:
@BlaineEXE  wrote:
> What is this version? Are we pinning to a Snyk version?
Good question!
Honestly, I was wondering as well, but could not get a proper explanation from the doc
https://docs.snyk.io/manage-risk/policies/the-.snyk-file ...

Since I was not sure about the effect of this parameter in the file, I had initiallt put the version of my local `snyk` cli tool.
I think 
this is also what happens if you use `snyk ignore` to create the `.snyk` file.


>Could we at least add a comment to explain what it's doing for ongoing maintenance?

yeah, we should add that after the fact once we have more clarity ...

Reply:
Maybe we can remove version from the file unless it breaks snyk. I think it's probably best to control the snyk version (assuming that's what this is) from the github action config that can get dependabot updates.

At least this example of the .snyk file appears to be whole and does not include the version: https://docs.snyk.io/scan-applications/snyk-code/snyk-code-and-your-repositories/excluding-directories-and-files-from-the-import-process#snytax-to-use-to-exclude-files-and-directories-from-snyk-code-testing